src/config/github-api.ts

```diff
@@ -1,12 +1,11 @@
 import { GitHubAPI } from "probot";
 import { getLogger } from "./logger";
-import { enhanceOctokit } from "./enhance-octokit";
 import { Options } from "probot/lib/github";
 
 export const GithubAPI = (options: Partial<Options> = {}): GitHubAPI => {
 	options.logger = options.logger || getLogger("config.github-api");
 	options.throttle = {
 		enabled: false
 	};
-	return enhanceOctokit(GitHubAPI(options as Options));
+	return GitHubAPI(options as Options);
```

Comment:
will be removing GitHubAPI in another PR since it would blow this one out.
Focus was `enhanceOctokit`